src/help/zaphelp/contents/ui/dialogs/manageaddons.html

```diff
@@ -29,6 +29,11 @@
 You can install any of the add-ons listed - typically add-ons are dynamically added to the ZAP UI
 as soon as you install them.
 
+<H2>Manual loading</H2>
+When debugging or working in a disconnected environment you may want to manually download and install an add-on. </br>
```

Comment:
- With debugging are you referring to ZAP/add-on development? or?
 - The slash in the `br` tag should be removed.